src/web/lib/components/ThemeBackgroundPicker/index.js

```diff
@@ -7,7 +7,7 @@ import Metrics from "../../../../lib/metrics";
 
 import "./index.scss";
 
-const bgImages = require.context("../../../../images/", false, /bg-.*\.png/);
+const bgImages = require.context("../../../../images/patterns/", false, /bg-.*\.svg/);
```

Comment:
I don't know enough about the internal magic of Webpack's `require.context()`, or if it's possible to save the /images/patterns/*.svg array in /src/lib/constants.js since it looks like we're repeating this line in 3 places in this PR.

¯\\\_(ツ)\_/¯

Reply:
this sounds like a @lmorchard question since webpack still seems mostly terrifyingly opaque to me...maybe file a separate issue?

Reply:
Actually, what might be a good idea is to move `bgImages` into `src/lib/themes.js` as a shared export since we use it both in the add-on and the website. But, I wouldn't block this PR to do it.